sympy/tensor/array/sparse_ndim_array.py

```diff
@@ -48,6 +45,11 @@ def __getitem__(self, index):
         if syindex is not None:
             return syindex
 
+        if isinstance(index, Integer):
+            index = Tuple(index)
```

Comment:
why not `tuple`?

Reply:
I was trying to cast an Integer to a tuple: 1 -> (1, ), but the problem is that the builtin `tuple` doesn't accept only one Integer because it is not iterable. However SymPy's `Tuple` can handle this case. 
An alternative would be: `(index, )`. Which one do you think is better? I was using `Tuple` because I think it can directly sympify the value.

Reply:
Go for `index = (index,)`.